proto/storage/cve.proto

```diff
@@ -148,7 +148,7 @@ message ImageCVE {
   google.protobuf.Timestamp snooze_start = 8 [deprecated = true];
   google.protobuf.Timestamp snooze_expiry = 9 [deprecated = true]; // @gotags: search:"CVE Snooze Expiry,hidden"
   // nvdcvss stores cvss score for a cve from NVD
-  float nvdcvss = 10; // @gotags: search:"NVD CVSS",store"
+  float nvdcvss = 10; // @gotags: search:"NVD CVSS,store"
```

Comment:
Does that quote change anything? The search field worked before right?

Reply:
yes it worked before with graphql query.